onemask/ArchitectureStudy/app/src/main/java/com/android/studyfork/ui/adapter/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package com.android.studyfork.ui.adapter
+
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.android.studyfork.ui.TickerListFragment
+
+/**
+ * created by onemask
+ */
+class ViewpagerAdapter(fragmentManager : FragmentManager) :
+    FragmentPagerAdapter(fragmentManager){
```

Comment:
deprecated 되어서 변경하는게 나을 것 같아요
```suggestion
    FragmentPagerAdapter(fragmentManager, BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
```

Reply:
해당 리뷰에 대하여 찾아 볼 수 있어서 더 좋았습니다. 
리뷰 감사합니다 : ) 수정하였습니다.
7d88bb1